### ISSUE TITLE
How to get children blocks ? 

### ISSUE THREAD

**Issue by j-barot** [minwook-shin/notion-database#16]
Hi, i'm looking to get children blocks of a specific page, i can see i should be able to do it but i don't find a way ^^' Thanks !

**Comment by minwook-shin** [comment]
hello, @j-barot  it doesn't. 
notion official documentation also you to use the block API instead of page. 
this package does not support the block API. (But, can create children block!)

ref : https://developers.notion.com/reference/retrieve-a-page

> Retrieves a [Page object](https://developers.notion.com/reference/page) using the ID specified.

> Responses contains page properties, not page content. To fetch page content, use the [Retrieve block children](https://developers.notion.com/reference/get-block-children) endpoint.

I will let you know when finished developing feature.

**Comment by minwook-shin** [comment]
```python
B = Block(integrations_token=NOTION_KEY)
for j in D.result["results"]:
    B.retrieve_block(block_id=j["id"], is_children=True)
    pprint.pprint(B.result)
```
coming soon, (on 1.1.0 milestone ) :)

**Comment by minwook-shin** [comment]
@j-barot  released version 1.1.0, which add this feature.
https://github.com/minwook-shin/notion-database/releases/tag/1.1.0
```
pip install notion-database==1.1.0
```
Here an example.
https://github.com/minwook-shin/notion-database/blob/1.1.0/page_content_example.py

**Comment by j-barot** [comment]
Wow thanks ! I'll update and using it asap !
